src/DataClasses/property.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+class Property():
+	''' 
+	Constructor
+	Parameters: address (dict), residenceType(string)
+	'''
+	def __init__(self, address, residenceType):
+		self.address = address
+		self.residenceType = residenceType
```

Comment:
If it is a multiple unit property, I think we would also want to store how many units there are.